Add illustrations12 (#20)
Anmol-Baranwal/Awesome-Illustrations-4Projects: README.md

    - [Bam 3D](https://bamdesign.gumroad.com/l/bamillu)
    - [Big Heads](https://bigheads.io/)
    - [Blues](https://www.getillustrations.com/illustration-pack/blues-dual-tone-free-illustrations)
    - [Bottts Generator](https://www.botttsgenerator.com/)
    - [Bzzzt](https://www.ls.graphics/illustrations/bzzzt)
    - [Casual Life 3D](https://www.figma.com/community/file/1118811400396348161)
    - [Charco](https://www.charco.design/charco)
    - [Christmas](https://iconscout.com/illustration-pack/christmas-2333)
    - [Christmas 3D](https://christmas.wannathis.one/)
    - [Chubbs Character](https://drawkit.com/product/chubbs-character-illustrations)
    - [Control](https://craftwork.gumroad.com/l/ZWiR)
    - [Daily 50](https://www.getillustrations.com/illustration-pack/download-free-daily-illustrations)
    - [Delesign](https://delesign.com/free-designs/graphics/)
    - [Diversity Avatars](https://www.netguru.com/blog/inclusive-design-how-to-start)
    - [Doodle Ipsum](https://doodleipsum.com/)
    - [Drawer Avatar](https://drawer.design/products/avatar-library/)
    - [Ecology & Environment](https://drawkit.com/product/ecology-environment-illustrations)

>     - [Finn Character](https://drawkit.com/product/fun-playful-finn-character-illustrations)

Reviewer: ```suggestion
    - [Finn Character](https://drawkit.com/product/finn-character-illustrations)
```

| <a href="https://www.figma.com/community/file/990218169140929445"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643886-c5879237-c85b-488b-9860-3ef800568768.png' alt="3D_Cartoon_free_illustrations" /></a> | [3D Cartoon Hands](https://www.figma.com/community/file/990218169140929445)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Lionel Taurus                                                    |
| <a href="https://www.kapwing.com/404-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917308-fbec10ca-676f-4d62-a254-24407ef1440e.png' alt="404_free_illustrations" /></a> | [404 Illustrations](https://www.kapwing.com/404-illustrations)  <br><br> ✔ Free <br><br>Format <br> • PNG<br><br> ✌️Author <br> Kapwing                                                                                         |

> | <a href="https://drawkit.com/product/animal-pets-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193481250-fe920ee8-13cf-4de8-a416-1e834256cb40.png' alt="Animals_&_Pets_Illustrations" /></a> | [Animals & Pets](https://drawkit.com/product/animal-pets-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
change alt text to Animals_&_Pets_free_illustrations 
```

| <a href="https://www.botttsgenerator.com/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917328-17a42812-d4f2-44cc-8db2-6f6f0e82bca0.png' alt="Bottts_generator" /></a> | [Bottts Generator](https://www.botttsgenerator.com/)  <br><br> ✔ Free <br><br>Format <br> • SVG<br>• PNG<br><br> ✌️Author <br> Pablo Stanley                                                                                         |
| <a href="https://www.figma.com/community/file/1118811400396348161"><img width='700' src='https://user-images.githubusercontent.com/74038190/185668999-06d218a3-ad75-47ce-bb9a-7c03469d53a5.png' alt="casual-life-3d-free-illustrations" /></a> | [Casual Life 3D](https://www.figma.com/community/file/1118811400396348161)  <br><br> ✔ Free <br><br>Format: <br> • Figma<br><br> Author <br> Icons8                                                      | 

> | <a href="https://www.ls.graphics/illustrations/bzzzt"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479713-8d5b12fd-b366-43ca-aba1-ff24b0e78060.png' alt="Bzzt_3d_abstract_illustrations" /></a> | [Bzzzt](https://www.ls.graphics/illustrations/bzzzt)  <br><br> ✔ Free  <br><br>Format <br> • Figma<br> • PNG<br> • SVG<br> • Sketch<br> • Adobe XD<br> • Adobe Illustrator<br> • Affinity<br><br> ✌️Author <br> LS Graphics |

Reviewer: ```suggestion
change alt text to Bzzt_3d_abstract_free_illustrations
```

| <a href="https://iconscout.com/illustration-pack/christmas-2333"><img width='700' src='https://user-images.githubusercontent.com/74038190/184642584-532d5bea-2a3b-490f-a99c-a5fb986496ee.png' alt="Christmas_free_illustrations" /></a> | [Christmas](https://iconscout.com/illustration-pack/christmas-2333)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • Sketch<br> • EPS<br><br> ✌️Author <br> Iconscout                                    |
| <a href="https://christmas.wannathis.one/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184478914-e0edf39e-aee6-46c9-a953-00749d471cc5.png' alt="Christmas 3D_free_illustrations" /></a> | [Christmas 3D](https://christmas.wannathis.one/)  <br><br> ✔ Free <br><br>Format <br>• PNG<br><br> ✌️Author <br>Wannathis                                                                                              |

> | <a href="https://drawkit.com/product/chubbs-character-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193478987-57701db5-dc94-4adc-af94-6d849b9beb8a.png' alt="Chubbs_Character_Illustrations" /></a> | [Chubbs_Character_Illustrations](https://drawkit.com/product/chubbs-character-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
change the title from Chubbs_Character_Illustrations to Chubbs Character
add the Format of Adobe Illustrator
change alt text to Chubbs_Character_free_illustrations
```

| <a href="https://doodleipsum.com/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643856-ea39f8f7-6c5d-45a5-a474-65f38063594b.png' alt="Doodle_Ipsum_free_illustrations" /></a> | [Doodle Ipsum](https://doodleipsum.com/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Blush |
| <a href="https://drawer.design/products/avatar-library/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184863865-4c6200a4-cd20-4695-a330-9ae0f9735b55.png' alt="Drawer_Avatar_free_illustrations" /></a> | [Drawer Avatar](https://drawer.design/products/avatar-library/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br><br> ✌️Author <br> Drawer                                                                  |

> | <a href="https://drawkit.com/product/ecology-environment-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479444-40aa7a75-b8df-4736-a4f0-8001cd362f5b.png' alt="ecology_&_environment_illustrations" /></a> | [Ecology & Environment](https://drawkit.com/product/ecology-environment-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
add the format of Figma
change alt text to ecology_&_environment_free_illustrations
```

> | <a href="https://drawkit.com/product/fun-playful-finn-character-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479227-c995cc31-2074-43e0-84f6-006ab06b54d2.png' alt="Finn_Character_Illustrations" /></a> | [Finn Character](https://drawkit.com/product/fun-playful-finn-character-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
Insert the correct link https://drawkit.com/product/finn-character-illustrations
Add the format of Figma
Change alt text to Finn_Character_free_illustrations
```

| <a href="https://www.figma.com/community/file/882513090157409891"><img width='700' src='https://user-images.githubusercontent.com/74038190/184642192-b211e912-9796-4bbc-9613-40ef51639de6.png' alt="Hitu_free_illustrations" /></a> | [Hitu](https://www.figma.com/community/file/882513090157409891)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Mr.Biscuit & YingMing                                                          |
| <a href="https://www.humaaans.com/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184915549-5dba6700-4d6e-4f4f-9d61-0976f1ccefea.png' alt="Humaaans_free_illustrations" /></a> | [Humaaans](https://www.humaaans.com/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• InVision Studio<br><br> ✌️Author <br> Humaaans                                                                             |

> | <a href="https://drawkit.com/product/influencer-content-creator-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193481009-6ee4d80d-ad58-492f-9e71-cf3772bae504.png' alt="Influencer_&_Content_Creator_Illustrations" /></a> | [Influencer & Content Creator](https://drawkit.com/product/influencer-content-creator-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
change alt text to Influencer_&_Content_Creator_free_illustrations
```

| <a href="https://iradesign.io/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184916769-74432087-2bb0-4989-b547-c48c154b5082.png' alt="IRA_Design_free_illustrations" /></a> | [IRA Design](https://iradesign.io/)  <br><br> ✔ Free <br><br>Format <br> • PNG<br>• SVG<br><br> ✌️Author <br> IRA Design |
| <a href="https://illlustrations.co/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184864144-c0b307d6-3284-4533-9fd9-94c607ff42d4.png' alt="Open_Source_free_illustrations" /></a> | [Illustrations.co](https://illlustrations.co/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI<br><br> ✌️Author <br> Vijay verma                                                             |

> | <a href="https://drawkit.com/product/larry-character-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193478852-c46e4b08-2467-4f7d-a968-d5c1c9720d79.png' alt="Larry_Character_Illustrations" /></a> | [Larry Character](https://drawkit.com/product/larry-character-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br>• PNG<br>• Figma<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
change alt text to Larry_Character_free_illustrations
```

| <a href="https://www.kapustin.co/product/lifestyle"><img width='700' src='https://user-images.githubusercontent.com/74038190/184643879-032ab493-1c9c-43c9-aa64-1b315dd7f6ec.png' alt="Lifestyle_free_illustrations" /></a> | [Lifestyle](https://www.kapustin.co/product/lifestyle)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• AI<br>• SVG<br>• AI<br><br> ✌️Author <br> kapustin.co                                   |
| <a href="https://www.getillustrations.com/illustration-pack/loomies-free-vector-illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808021-dec8c17c-b35a-404f-b76c-b7baf2bd19c5.png' alt="Loomies_free_illustrations" /></a> | [Loomies 39](https://www.getillustrations.com/illustration-pack/loomies-free-vector-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• InVision Studio<br><br> ✌️Author <br> Getillustrations                                                   |

> | <a href="https://drawkit.com/product/love-dating-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193481353-e788ff3c-b8f5-4986-8da6-e3e5d957822e.png' alt="Love_&_Dating_Illustrations" /></a> | [Love & Dating](https://drawkit.com/product/love-dating-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br> • Figma<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
change alt text to Love_&_Dating_free_Illustrations
```

| <a href="https://lukaszadam.com/illustrations"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917833-5f31c70d-3d69-41ba-bd7e-c58257d409ef.png' alt="Lukaszadam_free_illustrations" /></a> | [Lukaszadam](https://lukaszadam.com/illustrations)  <br><br> ✔ Free <br><br>Format <br> • SVG<br>• PNG<br><br> ✌️Author <br> Lukaszadam                                                                              |
| <a href="https://www.manypixels.co/gallery"><img width='700' src='https://user-images.githubusercontent.com/74038190/184810217-55c81d5a-321f-401d-aa27-4864c8a197a6.png' alt="Manypixels_free_illustrations" /></a> | [Many Pixels](https://www.manypixels.co/gallery)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Getillustrations                             |

> | <a href="https://www.pixsellz.io/material-icons-library"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480907-db472e82-2421-4476-9f3f-df0db0667333.png' alt="Material_Icons_Library" /></a> | [Material Icons](https://www.pixsellz.io/material-icons-library)  <br><br> ✔ Free  <br><br>Format <br> • Figma<br> • Sketch<br><br> ✌️Author <br> Pixellz |

Reviewer: ```suggestion
change alt text to Material_Icons_free_Library
correct the name of the author to Pixsellz
```

| <a href="https://github.com/MariaLetta/mega-doodles-pack/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917535-88553471-26eb-4bf2-a532-b719341a66c8.png' alt="Mega_Doodles_Pack_free" /></a> | [Mega Doodles Pack](https://github.com/MariaLetta/mega-doodles-pack/)  <br><br> ✔ Free <br><br> |

> | <a href="https://drawkit.com/product/mental-health-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193481139-ef3f5fca-4514-4fed-8b88-8ad6ab27580b.png' alt="Mental_Health_Illustrations" /></a> | [Mental Health](https://drawkit.com/product/mental-health-illustrations)  <br><br> ✔ Free  <br><br>Format <br> • SVG<br> • PNG<br><br> ✌️Author <br> DrawKit |

Reviewer: ```suggestion
change alt text to Mental_Health_free_illustrations
```

| <a href="https://www.streamlinehq.com/illustrations/free-illustrations-bundle"><img width='700' src='https://user-images.githubusercontent.com/74038190/184809664-a30ad560-c9e5-4e81-b3df-30113a624952.png' alt="Streamlinehq_free_illustrations" /></a> | [Streamline HQ](https://www.streamlinehq.com/illustrations/free-illustrations-bundle)  <br><br> ✔ Free <br><br>Format <br>• PNG<br>• SVG<br>• PDF<br><br> ✌️Author <br> Streamlinehq            |
| <a href="https://www.figma.com/community/file/883778082594341562"><img width='700' src='https://user-images.githubusercontent.com/74038190/184642936-50c41348-3663-436e-bab6-5233d306b960.png' alt="Surface_free_illustrations" /></a> | [Surface 75](https://www.figma.com/community/file/883778082594341562)  <br><br> ✔ Free <br><br>Format <br> • Figma<br><br> ✌️Author <br> Treetop                                                             |

> | <a href="https://drawkit.com/product/team-work-illustrations"><img width='700' src='https://user-images.githubusercontent.com/92253071/193478432-a4e0fd47-1633-4b39-a680-d5f936b96753.png' alt="Team_work_illustrations" /></a> | [Team Work](https://drawkit.com/product/team-work-illustrations)  <br><br> ✔ Free <br><br>Format <br> • Figma<br> • PNG<br>• SVG<br><br> ✌️Author <br> Drawkit                                                             |

Reviewer: ```suggestion
change alt text to Team_work_free_illustrations                                                 |
```

| <a href="https://www.abstrakt.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184916241-46cc2949-d749-4695-9813-05569b88af99.png' alt="Abstrakt_freemium_illustrations" /></a> | [Abstrakt](https://www.abstrakt.design/)  <br><br> ✨ Freemium <br><br>Format <br> • PNG<br>• SVG<br><br> ✌️Author <br> Abstrakt.design                                                                                                                |
| <a href="https://blush.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808594-ada33e0a-022f-4b46-b595-fb816b6954fe.png' alt="Blush_freemium_illustrations" /></a> | [Blush](https://blush.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Small PNG<br>• Medium PNG<br>• Large PNG<br>• SVG<br><br> ✌️Author <br> Blush Design                                                                                         |

> | <a href="https://www.ls.graphics/illustrations/ciank-illustration-kit"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480180-8b180a05-2fd2-42d1-a6e3-88206018d7fd.png' alt="Ciank!_illustration_kit" /></a> | [Ciank!](https://www.ls.graphics/illustrations/ciank-illustration-kit)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br> • Sketch<br> • Adobe Illustrator<br> • Affinity<br> • Adobe XD<br> • PNG<br> • SVG<br><br> ✌️Author <br> LS Graphics |

Reviewer: ```suggestion
change the title from Ciank! to Ciank
change alt text to Ciank_freemium_illustration_kit
```

| <a href="https://icons8.com/l/christmas-illustrations/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184641769-0aa18a5c-2f4a-4523-9d44-fc715b62aa18.png' alt="Merry_Christmas_freemium_illustrations" /></a> | [Merry Christmas](https://icons8.com/l/christmas-illustrations/)  <br><br> ✨ Freemium <br><br>Format <br> • Large PNG<br>• SVG<br> • GIF<br> • PSD<br> • JSON<br> • After Effects<br><br> ✌️Author <br> Icons8           |
| <a href="https://www.ls.graphics/illustrations/pof-pof"><img width='700' src='https://user-images.githubusercontent.com/74038190/184863554-c0eef9aa-022a-433d-af05-c1281aa15a89.png' alt="Pof-Pof_freemium_illustrations" /></a> | [Pof-Pof](https://www.ls.graphics/illustrations/pof-pof)  <br><br> ✨ Freemium mium<br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI<br><br> ✌️Author <br> ls.graphics                                           |

> | <a href="https://www.kapustin.co/seoul"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480665-4f035a53-902a-4ecd-9b37-a174e00c8183.png' alt="Seoul_illustrations" /></a> | [Seoul](https://www.kapustin.co/seoul)  <br><br> ✨ Freemium  <br><br>Format <br> • Adobe Illustrator<br> • EPS<br> • PDF<br> • SVG<br><br> ✌️Author <br> Kapustin.co |

Reviewer: ```suggestion
change alt text to Seoul_freemium_illustrations
add formats: Figma, Sketch, Small PNG (e.g. see in Blush)
```

| <a href="https://usesmash.com/"><img width='700' src='https://user-images.githubusercontent.com/74038190/190926129-89598235-c80e-4cf1-84c8-51cfcd06dafc.png' alt="Smash_free_illustrations" /></a> | [Smash](https://usesmash.com/)  <br><br> ✔ Free <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI <br><br>                                                                                                                           |
| <a href="https://stubborn.fun/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184917527-3f5dde24-889b-4d1b-a948-17d38e7476b9.png' alt="Stubborn_freemium_illustrations" /></a> | [Stubborn](https://stubborn.fun/)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Getillustrations                                                                    |

> | <a href="https://www.kapustin.co/tokyo"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480612-caec3f3f-eda1-42a9-9e00-6a91ebca69b0.png' alt="Tokyo_Illustrations" /></a> | [Tokyo](https://www.kapustin.co/tokyo)  <br><br> ✨ Freemium  <br><br>Format <br> • Adobe Illustrator<br> • EPS<br> • PDF<br> • SVG<br><br> ✌️Author <br> Kapustin.co |

Reviewer: ```suggestion
change alt text to Tokyo_freemium_illustrations
add formats: Figma, Sketch, Small PNG
```

| <a href="https://www.ls.graphics/illustrations/whoosh"><img width='700' src='https://user-images.githubusercontent.com/74038190/184864391-7642a6f3-7698-4390-a135-bc5793953c83.png' alt="Whoosh_freemium_illustrations" /></a> | [Whoosh](https://www.ls.graphics/illustrations/whoosh)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• SVG<br>• AI<br><br> ✌️Author <br> ls.graphics                                             |

> | <a href="https://www.ls.graphics/illustrations/yeeow"><img width='700' src='https://user-images.githubusercontent.com/92253071/193479942-5f1d44b5-55a3-49b9-86fa-9e7dafabede5.png' alt="Yeeowq!_illustration_kit" /></a> | [Yeeow!](https://www.ls.graphics/illustrations/yeeow)  <br><br> ✨ Freemium  <br><br>Format <br> • Affinity<br> • Figma<br> • Adobe Illustrator<br> • Sketch<br> • SVG<br> • PNG<br><br> ✌️Author <br> LS Graphics |

Reviewer: ```suggestion
change the title from Yeeow! to Yeeow
change alt text from Yeeowq!_illustration_kit to Yeeow_freemium_illustration_kit
```

| <a href="https://blush.design/"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808594-ada33e0a-022f-4b46-b595-fb816b6954fe.png' alt="Blush_freemium_illustrations" /></a> | [Blush](https://blush.design/)  <br><br> ✨ Freemium <br><br>Format <br> • Small PNG<br>• Medium PNG<br>• Large PNG<br>• SVG<br><br> ✌️Author <br> Blush Design                                                                                         |
| <a href="https://www.ls.graphics/illustrations/ciank-illustration-kit"><img width='700' src='https://user-images.githubusercontent.com/92253071/193480180-8b180a05-2fd2-42d1-a6e3-88206018d7fd.png' alt="Ciank!_illustration_kit" /></a> | [Ciank](https://www.ls.graphics/illustrations/ciank-illustration-kit)  <br><br> ✨ Freemium  <br><br>Format <br> • Figma<br> • Sketch<br> • Adobe Illustrator<br> • Affinity<br> • Adobe XD<br> • PNG<br> • SVG<br><br> ✌️Author <br> LS Graphics |

> | <a href="https://www.ls.graphics/illustrations/clak"><img width='700' src='https://user-images.githubusercontent.com/74038190/184808599-e430ada2-18af-4581-bfe7-c70b2b832853.png' alt="Ciank_freemium_illustration_kit" /></a> | [Clak](https://www.ls.graphics/illustrations/clak)  <br><br> ✨ Freemium <br><br>Format <br> • Figma<br>• Sketch<br>• Adobe XD<br>• PNG<br>• SVG<br>• AI<br><br> ✌️Author <br> Ls Graphics                                               |

Reviewer: This was okay, please reverse this because I think you mistakenly changed `Clak` instead of `Ciank Illustration Kit` & please change the Ciank illustration above the clak

Author: Apologies - am making amendments now

- Have reverted to the original alt text for clack
- Have updated the Ciank alt text
- Have updated the name to Ciank in the menu
